.github/workflows/scala3-build.yml

```diff
@@ -0,0 +1,35 @@
+name: Build Akka with Scala 3
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags-ignore: [ v.* ]
+
+jobs:
+  compile-and-test-with-scala3:
+    name: Compile and test with Scala 3
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Checkout GitHub merge
+        if: github.event.pull_request
+        run: |-
+          git fetch origin pull/${{ github.event.pull_request.number }}/merge:scratch
+          git checkout scratch
+
+      - name: Set up JDK 11
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11.0-9
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v5
+
+      - name: Compile all code with fatal warnings for Java 11, Scala 3
+        run: sbt -jvm-opts .jvmopts-travis -Dakka.build.scalaVersion=3.0 akka-actor-tests/test akka-actor-testkit-typed/test akka-actor-typed/compile akka-actor-typed-tests/test akka-discovery/test akka-pki/test akka-protobuf/test akka-protobuf-v3/test akka-slf4j/test akka-stream/test akka-stream-tests-tck/test akka-coordination/test akka-serialization-jackson/test:compile akka-testkit/test akka-stream-testkit/test akka-remote/compile
```

Comment:
```suggestion
        run: |
          sbt -jvm-opts .jvmopts-travis -Dakka.build.scalaVersion=3.0 \
            akka-actor-tests/test \
            akka-actor-testkit-typed/test \
            akka-actor-typed/compile \
            akka-actor-typed-tests/test \
            akka-discovery/test \
            akka-pki/test \
            akka-protobuf/test \
            akka-protobuf-v3/test \
            akka-slf4j/test \akka-stream/test \
            akka-stream-tests-tck/test \
            akka-coordination/test \
            akka-serialization-jackson/test:compile \
            akka-testkit/test \
            akka-stream-testkit/test \
            akka-remote/compile
```

Reply:
Off-topic I keep forgetting about:

 - It may be about time that we renamed `.jvmopts-travis` to `.jvmopts-ci` or something.

Reply:
> It may be about time that we renamed .jvmopts-travis to .jvmopts-ci or something

Good idea. Will do it.